pkg/services/alerting/test_notification.go

```diff
@@ -54,7 +55,7 @@ func createTestEvalContext(cmd *NotificationTestCommand) *EvalContext {
 		State:       m.AlertStateAlerting,
 	}
 
-	ctx := NewEvalContext(context.Background(), testRule)
+	ctx := NewEvalContext(context.Background(), testRule, &setting.Cfg{})
```

Comment:
Not sure if this is for testing only and whether a real Cfg needs to be propagated here.